manifests/server.pp

```diff
@@ -62,7 +63,6 @@
 
   file { "${config_dir}/rules":
     ensure => 'directory',
-    owner  => $user,
```

Comment:
@anarcat Why did you remove the owner? I am pretty sure this is set on purpose.

Reply:
same answer.